left:
  pose:
    preHandoff:
      position:
        x: 0.1
        y: -0.4
        z: 0.3
      orientation:
        x: 0.5
        y: 0.5
        z: 0.5
        w: 0.5

    handoff:
      position:
        x: 0.19
        y: -0.4
        z: 0.3
      orientation:
        x: 0.5
        y: 0.5
        z: 0.5
        w: 0.5

    pickup:
      position:
        x: -0.3
        y: -0.09
        z: 0.1
      orientation:
        x: -0.999911
        y: 0.006463
        z: -0.007255
        w: -0.009191

    0g_pickup:
      position:
        x: -0.3
        y: -0.09
        z: 0.19
      orientation:
        x: -1
        y: 0
        z: 0
        w: 0

    preThrow:
      position:
        x: 0
        y: -0.4
        z: 0.45
      orientation:
        x: 0.5
        y: -0.5
        z: -0.5
        w: 0.5

    postThrow:
      position:
        x: -0.2
        y: -0.4
        z: 0.45
      orientation:
        x: 0.5
        y: -0.5
        z: -0.5
        w: 0.5

    

    preGrasp:
      position:
        x: -0.3
        y: -0.1
        z: 0.4
      orientation:
        x: -0.999911
        y: 0.006463
        z: -0.007255
        w: -0.009191

    postGrasp:
      position:
        x: -0.25677
        y: -0.0545
        z: 0.25
      orientation:
        x: -0.999911
        y: 0.006463
        z: -0.007255
        w: -0.009191
      
hardware:
  pose/pickup:
    position:
      x: 0.3
      y: 0.1
      z: 0.1
    orientation:
      x: 0
      y: -1
      z: 0
      w: 0

  pose/preGrasp:
    position:
      x: 0.3
      y: 0.1
      z: 0.4
    orientation:
      x: 0
      y: -1
      z: 0
      w: 0



right:
  pose/preHandoff:
    position:
      x: 0.38
      y: -0.4
      z: 0.35
    orientation:
      x: 0.5
      y: -0.5
      z: -0.5
      w: 0.5

  pose/handoff:
    position:
      x: 0.28
      y: -0.4
      z: 0.35
    orientation:
      x: 0.5
      y: -0.5
      z: -0.5
      w: 0.5

  pose/pickup:
    position:
      x: 0.834
      y: -0.04
      z: 0.15
    orientation:
      x: 0.679
      y: -0.733
      z: -0.042
      w: 0.033

gripper:
  open:
    -0.4
  close:
    0.6981
